domains.md

```diff
@@ -3,7 +3,7 @@ Domains, DNS conf and certificate
 
 YunoHost allows you to manage and serve several domains on the same server. For instance, you can host a blog and Nextcloud on a first domain `yolo.com`, and a web mail client on a second domain `swag.nohost.me`. Each domain is automatically configured to handle web services, mail services and XMPP services.
 
-Domains can be managed in the 'Domain' section of the webadmin, or through the `yunohost domain` category of the command line. Each time you add a domain, it is expected that you bought it (or own it) on a domain registrar, so you can manage the [DNS configuration](dns). The exception is the [domains `.nohost.me`, `.noho.st` and `ynh.fr`](/dns_nohost_me) which are free and can be directly integrated with YunoHost.
+Domains can be managed in the 'Domain' section of the webadmin, or through the `yunohost domain` category of the command line. Each time you add a domain, it is expected that you bought it (or own it) on a domain registrar, so you can manage the [DNS configuration](dns). The exception is the [`.nohost.me`, `.noho.st` and `ynh.fr` domains](/dns_nohost_me) which are paid for by the YunoHost Project, and can be directly integrated with YunoHost thanks to an automated dynDNS setup. (To limit costs and abuses, each instance may only have one of these domains setup at any given time).
```

Comment:
I was wondering about how this was funded. I would recommend it goes under it's own heading? It would be nice to have a separate section making clear the economic considerations of using this facility.

Reply:
well, AFAIN, funding is still "in works" within the project : NGO status is pending, and donations are collected through Liberapay, but no official budget/list of spendings has been set up...

Reply:
Ah, I just meant for the DynDNS stuff, not the entire project. Good to know though :)

Reply:
Hopefully we should be done with the bank account setup during the next 4 weeks and then we can open a real donation mechanism ;P